subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/ExcludeRuleMergingPerformanceTest.groovy

```diff
@@ -18,7 +18,9 @@ package org.gradle.performance.regression.corefeature
 
 import org.gradle.performance.AbstractCrossVersionPerformanceTest
 import org.gradle.performance.WithExternalRepository
+import spock.lang.Ignore
 
+@Ignore("Ignoring until it can be rebased as the fix for grdale/gradle#7050 introduces a small regression")
```

Comment:
@dansanduleac is this meant to be ignored?

Reply:
That's the way we deal with performance regression we have decided to accept as we do more work. Test has been re-enabled since then.